network/mounts.csv

```diff
@@ -15,7 +15,7 @@ TOD02,EN,West Lower Te Maari,-39.10304,175.67048,1406,WGS84,SO2 from Tongariro,2
 TOKR,VC,Karewarewa,-39.094442315,175.641203517,1207,WGS84,Images of Te Maari Crater.,2012-08-29T02:00:00Z,9999-01-01T00:00:00Z
 WHHB,VC,Whakatane Harbour,-37.949622999,177.002304198,10,WGS84,Images of White Island.,2001-05-21T13:00:10Z,2007-04-13T00:00:00Z
 WHOH,VC,Whakatane Observatory Hill,-37.956903132,177.000764155,100,WGS84,Images of White Island.,2006-03-29T05:32:07Z,9999-01-01T00:00:00Z
-WICF,VC,White Island Crater Floor,-37.526515575,177.189407876,10,WGS84,Images of White Island crater.,2009-10-28T00:45:02Z,9999-01-01T00:00:00Z
+WICF,VC,White Island Crater Floor,-37.52592,177.19322,10,WGS84,Images of White Island crater.,2009-10-28T00:45:02Z,9999-01-01T00:00:00Z
```

Comment:
not sure how far this is from the GNSS site (RGWC), but here's the coordinates of the GNSS antennas
`RGWC,CG,no,White Island Crater Floor,-37.524338674,177.192476273`

Reply:
100m or more away from GNSS site